src/addrman.cpp

```diff
@@ -1065,6 +1099,15 @@ int AddrManImpl::CheckAddrman() const
     if (nKey.IsNull())
         return -16;
 
+    int cnt_new{0};
+    int cnt_tried{0};
+    for (auto net : map_counts) {
+        cnt_new += net.second.first;
+        cnt_tried += net.second.second;
+    }
+    if (cnt_new != nNew || cnt_tried != nTried) {
+        return -20;
```

Comment:
Perhaps it's a good opportunity to start using a named variable? :)

Reply:
Do you mean getting rid of the return codes and return something like error strings instead? 
Hmm, not sure if this wouldn't better be a separate PR.

Reply:
ok - I think refactoring the codes might make sense, but if so I'd prefer to do this everywhere at once (and in a separate refactoring PR)